scripts/demo/demo_security_barrier_camera.sh

```diff
@@ -116,11 +91,8 @@ if ! command -v $python_binary &>/dev/null; then
     exit 1
 fi
 
-if [[ $DISTRO == "macos" ]]; then
-    "$pip_binary" install -r "$ROOT_DIR/../open_model_zoo/tools/downloader/requirements.in"
-else
-    sudo -E "$pip_binary" install -r "$ROOT_DIR/../open_model_zoo/tools/downloader/requirements.in"
-fi
+"$python_binary" -m pip install -U pip
+"$python_binary" -m pip install -r "$ROOT_DIR/../open_model_zoo/tools/downloader/requirements.in"
```

Comment:
use --user mode like on Windows.

Reply:
I have a great idea lets use one venv for all pip installation. Create it in the demo folder.